app/controllers/application_controller.rb

```diff
@@ -1,9 +1,17 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user, :author_name
+  def current_user
+    User.find_by_id session[:user_id]
+  end
 
- def hello
-    render text:"hi bud"
- end
+  def author_name (user_id)
+  	unless user_id.blank?
+	  author = User.find user_id
+      author.username
+  	else
+  	  '[unknonw]'
+  	end
+  end
```

Comment:
Can we :fire: this? If the foreign keys are set up right this shouldn't be necessary at all.